tests/admin_views/models.py

```diff
@@ -115,7 +115,7 @@ class CustomArticle(models.Model):
 
 
 class ModelWithStringPrimaryKey(models.Model):
-    string_pk = models.CharField(max_length=255, primary_key=True)
+    string_pk = models.CharField(max_length=150, primary_key=True)
```

Comment:
Not sure if 150 is intentional or arbitrary, but isn't 191 the max allowed length considering 4-byte characters and a maximum index of 767 bytes?